### PR TITLE
shifting s3 bucket for production data

### PR DESCRIPTION
**Pull request by elisehuard**

### CODE REVIEW

**Comment by acron0** [comment]
IIRC the reason we had to install docker 1.9 in circle over on [witan.ui](https://github.com/MastodonC/witan.ui/blob/master/circle.yml#L3) was to add build arg support, so we might need to do the same for this.

**Comment by acron0** [comment]
I also notice that we don't use the same aliasing when pushing witan.ui to docker (i.e. witan.app for prod or witan.app.staging for staging). I like this and we should do the same for witan.ui if this works.

File: run_witan_app.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# config
+cat <<EOF  > /root/.witan-app.edn
+{:cassandra-session {:host "cassandra-dcos-node.cassandra.dcos.mesos"
+                     :keyspace "witan"
+                     :replication 3}
+ :s3 {:bucket "witan-${ENVIRONMENT}-data"}}
```
Comment (sw1nn) [inline run_witan_app.sh]: if you replace `${ENVIRONMENT}` with `${ENVIRONMENT?not defined}` you'll get a 'ENVIRONMENT not defined' message and failure to start if `${ENVIRONMENT}` evaluates to `""` - this prevents wierdness

**Comment by sw1nn** [comment]
recommend the 'not defined' addition then :shipit:

**Pull request merged**